AxonDeepSeg/postprocessing.py

```diff
@@ -135,3 +135,23 @@ def generate_axon_numbers_image(centroid_index, x0_array, y0_array, image_size,
     image_array = np.asarray(number_image)
 
     return image_array.astype(np.uint8)
+
+def generate_and_save_colored_image_with_index_numbers(filename, axonmyelin_image_path, index_image_array):
```

Comment:
@Stoyan-I-A, I believe the tests for this new function (`generate_and_save_colored_image_with_index_numbers`) are missing from the tests. Could you write unit tests for this as well so that we can be sure that it works as expected?

Reply:
You're right, I will write a test for this function. It might require adding some new files to the `test_data `however.